app/utils/External.ts

```diff
@@ -5,6 +5,8 @@ import log from 'loglevel';
 import 'isomorphic-fetch';
 import { saveAs } from 'file-saver';
 import JSZip from 'jszip';
+import { stringify as queryStringify } from 'query-string';
```

Comment:
Minor:

I would call it

```suggestion
import { stringify as stringifyQueryString } from 'query-string';
```

as it sounds to me more like a function :slightly_smiling_face: